src/Symfony/Component/HttpKernel/DependencyInjection/AddCacheClearerPass.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Registers the cache clearers.
+ *
+ * @author Dustin Dobervich <ddobervich@gmail.com>
+ */
+class AddCacheClearerPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->hasDefinition('cache_clearer')) {
```

Comment:
same as for your other PR, target service id + collected tag name should be configurable through the constructor, defaulting to the ones used in frameworkbundle.